addon/components/faceted-search/style.scss

```diff
@@ -0,0 +1,55 @@
+/* Override in consuming application to customize appearance of search facets */
+
+.panel-default {
+    font-family: 'Open Sans' , 'Helvetica Neue' , sans-serif;
+    border-width: 0 ;
+
+    > .panel-heading {
+          background-color: rgba(40, 59, 73, 0.84);
+          color: white ;
+          display: flex;
+          align-items: center;
+          justify-content: center;
+          border-radius: 2px ;
+          font-size: 18px;
+    }
+}
+
+.ember-power-select-trigger {
+    background-color: #f5f5f5;
+    border-width: 1px;
+    border-color: #ccc;
+    border-radius: 0 ;
+    color: #333333 ;
+}
+
+.ember-power-select-placeholder {
+    color: #333333 ;
+}
+
+::-webkit-input-placeholder { /* WebKit, Blink, Edge */
+    color: #333333 ;
+}
+
+:-moz-placeholder { /* Mozilla Firefox 4 to 18 */
+    color: #333333;
+    opacity: 1;
+}
+
+::-moz-placeholder { /* Mozilla Firefox 19+ */
+    color: #333333;
+    opacity: 1;
+}
+
+:-ms-input-placeholder { /* Internet Explorer 10-11 */
+    color: #333333 ;
```

Comment:
I just copied this from Ember-Share.  I am not sure what you mean when you say 

> "missing the non-prefixed version of this pseudo-element"

but it doesn't look like you can put ::placeholder in the stylesheet.  It looks like you style placeholder text with the collection of vendor-prefixed selectors that are in this stylesheet.